### ISSUE TITLE
Table sync

### ISSUE THREAD

**Issue by Targma** [pgEdge/spock#12]
I setup database by backup/restore on two pgedge nodes with master-master configuration. I setup the subscription and it worked as intended. I tested sync when one DB is turned off for period of time while other received writes. This worked without problems.

Now if i remove subscription and create it again with same two DB, while one received writes, the new writes are replicated while the old writes are not. This could be due to background sync not completed yet, but based on given documentation and spock commands I am not sure if it is stuck or not. Database is 200mb when exported and is up for more then 30min, while some entries are still missing.

Is DB capable of syncing entries created before subscription was made ?
What can i do to sync rows.
Additionally some verbose logging would be appreciated.

All tables have PK uuid.

**Comment by luss** [comment]
This is a great question.   Let me have @cady &/or @tej give you specific advice.

**Comment by luss** [comment]
You need functioanlity in our ACE cli to synce table rows that are not part of the active replication set.  https://github.com/pgEdge/cli/blob/REL24_1/cli/ACE-README.md.  Specifically, table-diff and the repair-table

**Comment by donferi** [comment]
@luss how can we use `ace` on a local pgEdge platform?

I created two nodes `n1` and `n2` like getting started. Before creating the subscriptions I created a `users` table and added data in `n1` and `n2`.

I was expecting them  to have the same data after creating the replication sets similar to native postgres logical replication which do an initial sync.

Is there a way to trigger the initial sync with pgedge? If not how can I do the same with `table-diff` and `repair-table`? `ace` asks for a cluster name but I have no idea (and can't find docs) on what is the cluster name for my local one.
